includes/managers/icons.php

```diff
@@ -229,28 +227,6 @@ private static function is_font_icon_inline_svg() {
 		return Plugin::$instance->experiments->is_feature_active( 'e_font_icon_svg' );
 	}
 
-	/**
-	 * render_svg_symbols
-	 *
-	 */
-	public static function render_svg_symbols() {
```

Comment:
Mark as deprecated in order to avoid fatal errors, in case of using by a 3rd party plugin

Reply:
Done